tools/topology/sof-apl-pcm512x.m4

```diff
@@ -26,6 +27,9 @@ DEBUG_START
 # PCM1 ----> volume -----> iDisp1
 # PCM2 ----> volume -----> iDisp2
 # PCM3 ----> volume -----> iDisp3
+# PCM4 ----> volume -----> Media Playback 4
+# PCM5 <---- volume <----- DMIC0 (DMIC01)
+# PCM6 <---- volume <----- DMIC1 (DMIC16K)
```

Comment:
we've converged on 'DMIC' and "DMIC16kHz' for other topologies.

Reply:
I had DMIC16K initially, but I don't think it's available on Up^2 with the plugin board from @singalsu? So, I removed it, but forgot to remove the comment, will update. Or do we want it anyway? And "DMIC" instead of "DMIC01" everywhere?

Reply:
the same physical mic will provide PDM data to the Intel decimators, which can provide two outputs, such as 48kHz and 16 kHz. In other words there are no dedicated 16 kHz microphones and the availability of the 16 kHz path only depends on the Intel interface configuration.

and yes DMIC everywhere.

Reply:
aha, I thought the second DMIC would be the "secondary FIFO" as referred to in the dmic[] array in src/platform/intel/cavs/lib/dai.c. I'll try to enable 16kHz too then.

Reply:
hm, I see only 2 topologies using "kHz", all the rest use "16k"

Reply:
fixed now